app/models/manageiq/providers/storage_manager/storage_manager_mixin.rb

```diff
@@ -1,4 +1,4 @@
-module StorageManagerMixin
+module ManageIQ::Providers::StorageManager::StorageManagerMixin
```

Comment:
@roliveri I had assumed that we could call this StorageManagerMixin and have code for both the Cinder *and* Swift managers (and any others that come along) contained within, but if we want to be extremely "specific" with what we are being "specific" with... :-) I guess we can rename this one to CinderManagerMixin and add a separate file for the SwiftManagerMixin code.